Makefile

```diff
@@ -15,7 +15,24 @@ renew:
 	make clean && make run
 
 run:
-	docker run --name py-walletconnect-bridge -it -v $(shell pwd)/:/source/ -p 443:443 -p 80:80 py-walletconnect-bridge
+	docker run -it -v $(shell pwd)/:/source/ -p 443:443 -p 80:80 --name "py-walletconnect-bridge" py-walletconnect-bridge
 
 run_skip_certbot:
-	docker run --name py-walletconnect-bridge -it -v $(shell pwd)/:/source/ -p 443:443 -p 80:80 py-walletconnect-bridge --skip-certbot
+	docker run -it -v $(shell pwd)/:/source/ -p 443:443 -p 80:80 --name "py-walletconnect-bridge" py-walletconnect-bridge --skip-certbot
+
+update:
+	# save current state of DB and copy it to local machine
+	docker exec py-walletconnect-bridge redis-cli SAVE
+	docker cp py-walletconnect-bridge:/py-walletconnect-bridge/dump.rdb dump.rdb
+	docker container stop py-walletconnect-bridge
+	docker container rm py-walletconnect-bridge
+	
+	# start the container with `-d` to run in background
```

Comment:
Let's remove the existing image with the new one and build a new one before running the container
```
	# remove existing image and build new one
	docker rmi py-walletconnect-bridge
	docker build . -t py-walletconnect-bridge --build-arg branch=$(BRANCH)
```